test/index2.spec.js

```diff
@@ -33,6 +37,26 @@ describe('lintStaged', () => {
       },
       console
     )
-    expect(Listr.mock.calls[0][1]).toEqual({ dateFormat: false, renderer: 'verbose' })
+    expect(Listr.mock.calls[0][1]).toEqual({
+      concurrent: true,
+      dateFormat: false,
+      renderer: 'verbose'
+    })
+  })
+
+  it('should pass concurrent flag to Listr', async () => {
+    await lintStaged(
```

Comment:
Not sure what the best way to test this is. The tests don't actually seem to execute the `tasks`, so `Listr` never gets called with `concurrent`

Reply:
There are a couple of `unmocked` test files for `runAll` where it does actually run, maybe try there.